docs/source/federation-2/moving-to-federation-2.mdx

```diff
@@ -135,7 +135,7 @@ This definition identifies a schema as a true Federation 2 schema. Without this
 
 ### Mark all value types as `@shareable`
 
-By default in Federation 2, most schema fields are resolvable by only a _single subgraph_. In Federation 1, this is _not_ true for [value types](https://www.apollographql.com/docs/federation/value-types/):
+By default in Federation 2, most schema fields are resolvable by only a _single subgraph_. In Federation 1, this is _not_ true for [value types](https://www.apollographql.com/docs/federation/sharing-types/):
```

Comment:
Let's keep this as its existing URL, which points to the v1 docs (to show what value types are like in v1 in case the reader forgets)